perun-core/src/test/java/cz/metacentrum/perun/core/entry/VosManagerEntryIntegrationTest.java

```diff
@@ -854,6 +859,122 @@ public void getEnrichedVoById() throws Exception {
 		assertThat(enrichedVo.getParentVos()).containsOnly(parent);
 	}
 
+	public void addMemberVoNoExistingMember() throws Exception {
```

Comment:
`@Test ` is missing.

Reply:
added and also added test `removeMemberVoStartsParentVoLifecycle` to check that overtaking lifecycle by parent vo works